api/DocumentOrShadowRoot.json

```
@@ -244,7 +246,8 @@
               "version_added": "6.0"
             },
             "webview_android": {
-              "version_added": "53"
+              "version_added": "53",
+              "notes": "Before Chrome 66, this method returned <code>null</code> when the element was a child of a host node. See Chromium issue <a href='https://crbug.com/'>759947'</a> for details."
```

Please adapt note text to the browser it applies to (or omit the name of the browser)

```suggestion
              "notes": "Before WebView 66, this method returned <code>null</code> when the element was a child of a host node. See <a href='https://crbug.com/759947'>issue 759947</a>."
```